static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js

```diff
@@ -0,0 +1,82 @@
+define([
+    'lodash/utilities/template',
+    'commercial/modules/user-features',
+    'common/modules/commercial/contributions-utilities',
+    'lib/config',
+    'raw-loader!common/views/epic-for-brexit-cohort.html',
+    'svgs/icon/quote.svg',
+    'common/modules/commercial/acquisitions-epic-testimonial-parameters'
+], function (
+    template,
+    userFeatures,
+    contributionsUtilities,
+    config,
+    epicForBrexitCohortTemplate,
+    quoteSvg,
+    acquisitionsEpicTestimonialParameters
+) {
+
+    function hasBrexitTag(){
+        var tags = config.page.keywordIds.concat(config.page.nonKeywordTagIds);
+        return tags.indexOf('politics/eu-referendum') > -1;
+    }
+
+    function worksWellWithPageTemplate() {
```

Comment:
this is included as part of the standard `canRun` check in contributions utilities. If you remove `overrideCanRun` and still provide a `canRun` function that includes your brexit tag check, you'll get the page template check (and some other stuff) for free